Reflections/Week_5_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* They're not complete to the degree where I would want them to be but I have added more enemies and I have also made more rooms. I did refactor my Character and Fight class that made it more generic so that moving forward I'll be able to write my code faster and more effciently. I also updated the layout map for the 
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+*  I think the challenge I faced was not properly pacing myself but moving forward I will try to tackle my goals in more managable chunks.
```

Comment:
I agree with this! Proper pacing is important (proper use of LEAN and Agile helps with this a lot!)